src/main/java/view/HomeView.java

```diff
@@ -11,19 +11,125 @@ public class HomeView extends JPanel {
     private String viewName = "home";
     private HomeController homeController;
     private JButton starChart;
+    private JButton moonPhase;
     private JButton events;
+    private JButton neo;
     private JButton apod;
-    private JButton moonPhase;
     private JButton geocoder;
 
     public HomeView() {
         final JLabel title = new JLabel("Home Screen");
         title.setAlignmentX(Component.CENTER_ALIGNMENT);
 
         // Star Chart Button
```

Comment:
this comment can be deleted as line 24 semantically shows that there is a star chart button being created!